portafly/portafly_api.yml

```diff
@@ -68,35 +88,147 @@ components:
           type: integer
       example:
         id: 1
-        created_at: 2019-10-18T05:13:26Z
-        updated_at: 2019-10-18T05:13:27Z
+        created_at: "2019-10-18T05:13:26.000Z"
+        updated_at: "2019-10-18T05:13:27.000Z"
         state: approved
         org_name: Umbrella Corp.
         admin_name: Oswell E. Spencer
         apps_count: 3
+    AccountUser:
+      title: Root Type for accountUserData
+      description: A simplified format for an account user data.
+      type: object
+      properties:
+        id:
+          description: The user ID number
+          type: integer
+        name:
+          description: Name of the user
+          type: string
+        email:
+          description: Email of the user
+          type: string
+        role:
+          description: Role (admin/member) of the user
+          type: string
+        is_current_user:
+          description: Is user the currently logged in user?
+          type: boolean
+        can_see_users:
+          description: Can the user see other users?
+          type: boolean
+        authenticated_through_sso:
+          description: Is the user authenticated through SSO?
```

Comment:
Where are the descriptions coming from? Might be good to revise them. If these are new, would be better to be consistent in the redaction, (I.E: "the user / user / User / `User`)

Reply:
They are new, yes we should revise them